docs/articles/documentation/using-testcafe/programming-interface/runner.md

```diff
@@ -201,13 +201,14 @@ createTestCafe('localhost', 1337, 1338)
 Enables TestCafe to take screenshots of the tested webpages.
 
 ```text
-screenshots(path [, takeOnFails]) → this
+screenshots(path [, takeOnFails, pathPattern]) → this
 ```
 
 Parameter                  | Type    | Description                                                                   | Default
 -------------------------- | ------- | ----------------------------------------------------------------------------- | -------
-`path`                     | String  | The path to which the screenshots are saved.
+`path`                     | String  | The root path where the screenshots are saved. The relative path is specified by the default [path patterns](../command-line-interface.md#path-patterns) or by the method's `screenshotPathPattern` parameter.
```

Comment:
The relative path is specified by the default [path patterns](../command-line-interface.md#path-patterns) or by the method's `screenshotPathPattern` parameter. -> The default [path patterns](../command-line-interface.md#path-patterns) or the method's `screenshotPathPattern` parameter specifies the relative path.

Reply:
First we give information about the root path. Then we describe how to specify the relative path. And I think we should start this sentence from "The relative path is specified" as in my version. Why you revert this sentence?

Reply:
I think that we can use passive voice here. If we use Dirk's variant "The root path where the screenshots are saved.  The default path patterns or the method's screenshotPathPattern parameter specifies the relative path.", users can think that default path patterns relates to the root path. 

What do you think? @VasilyStrelyaev @AndreyBelym

Reply:
The default path patterns define the path to the base directory where screenshots are saved. You can use the screenshotPathPattern parameter to override these patterns.

Reply:
We should avoid **-ing**, so you can't use **using**. Passive voice is wordy, and you can avoid it when the thing that is performing the task is clear, in this case, the default path pattens.

Reply:
No!
The 'path' parameter specifies the base path to which the screenshots are saved. The relative path to a screenshot path is defined by the default path patters. You can override the default patterns with a custom pattern using the screenshotPathPattern parameter.

Reply:
No? Why do you ask for my review if you just insist on your variant? Ray said we should not use **-ing**, and 'This is performed by that' (5 words) -> 'That performs this' (3 words). Why do you want to use passive voice here?

Reply:
In the last rules from Ray, there are no rules about Passive Voice. He uses Passive Voice in his examples:
![image](https://user-images.githubusercontent.com/12034505/43004980-115d49ba-8c3a-11e8-9e8a-f584702fae55.png)

In our situation, I think we can use Passive Voice as it is better to understand a concepts about root path and relative path.

Reply:
That is a bad example. He uses passive voice because there is no alternative. If this sentence read "The capability to focus a grid row is not supported by the Grid", then it should be "The Grid does not support..." In the sentence you use as your justification, there is no 'by something'.

Reply:
> The path to the base directory where screenshots are saved. The default path patterns specify the actual/relative path to a screenshot file.

In this sentence, `the default path patterns` is 'new' information. If we leave it at the beginning, it would look as if we spoke about something already familiar to the reader, while it is not so. To my taste, it is natural to put it to the end, linking (again) the first and second sentences with `path to the base directory` vs `the actual path to a screenshot file`.

Reply:
I really do not believe users can easily understand 'a' concepts better because you use passive voice.

Reply:
Ray did not mention passive voice explicitly, but it falls under point no.5:
![image](https://user-images.githubusercontent.com/26871970/43005911-ea76fcb2-8c3c-11e8-8d4a-12ef6e6f9592.png)